app/views/layouts/_navbar.html.erb

```diff
@@ -18,7 +18,10 @@
     <% if signed_in? %>
       <span class="nav-link" data-testid="signin-name">
         <div class="mx-1">
-          <%= current_user.identities.empty? ? current_user.email : current_user.identities.first.name %>
+          <%= link_to edit_user_registration_path do %>
```

Comment:
we probably want this behind a toggle until it's ready to roll? otherwise if we merge this in we will need to go live with it?

Reply:
yep, great idea, will Flipper it!